packages/browser-repl/scripts/sync-to-compass.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-console */
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const child_process = require('child_process');
+const { debounce } = require('lodash');
```

Comment:
Any particular reason this isn't ESM?

Reply:
Ahhh - it was copied from the `sync-dist-to-mms.js` file.

Reply:
Having this typescript would make me happy. Could do it another time though.

Reply:
I copy/pasted it and it is just a developer productivity script. We can improve it whenever.